cypress/component/rich-text/RichTextEditor.Tracking.spec.ts

```diff
@@ -305,14 +307,13 @@ describe('Rich Text Editor - Tracking', { viewportHeight: 2000 }, () => {
 
     it('tracks insert table', () => {
       insertTable();
-
-      richText.expectTrackingValue([insertTableAction()]);
+      cy.get('@onAction').should('be.calledWith', ...insertTableAction());
```

Comment:
Nit: Those two assertions are not equal. The existing assertion checks if we get correct event and only the correct event.

The `be.calledWith` doesn't handled the case if we fire the correct tracking event + some nonsense that shouldn't be there.